test/units/module_utils/facts/system/distribution/fixtures/lmde_6.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Linux Mint Debian Edition 6",
+    "distro": {
+        "codename": "faye",
+        "id": "linuxmint",
+        "name": "LMDE",
+        "version": "6",
+        "version_best": "6",
+        "lsb_release_info": {},
+        "os_release_info": {
+            "pretty_name": "LMDE 6 (faye)",
+            "name": "LMDE",
+            "version_id": "6",
+            "version": "6 (faye)",
+            "version_codename": "faye",
+            "id": "linuxmint",
+            "home_url": "https://www.linuxmint.com/",
+            "support_url": "https://forums.linuxmint.com/",
+            "bug_report_url": "http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/",
+            "privacy_policy_url": "https://www.linuxmint.com/",
+            "id_like": "debian",
```

Comment:
we might want to factor `id_like` into the general code .. if this is a standard, we might be able to get rid of the hardcoded lists

Reply:
I am afraid that we can not do that since different distributions have different logic for implementing this.